src/main/java/uk/gov/hmcts/probate/service/tasklist/TaskStateRenderer.java

```diff
@@ -274,6 +284,16 @@ private String getLinkUrlTemplate(TaskListState taskListState, String willType)
             ihtEstate207 = "<li>" + IHT_ESTATE_207_TEXT + "</li>";
         }
         keyValue.put("ihtEstate207", ihtEstate207);
+        String admonWillRenForms = "";
```

Comment:
Tjere is actually a bunch of stuff like this I have setup as service methods in DTSPB-2086, new LinkFormatterService... maybe its better to push this AFTER that JIRA?